tests/test_scenario_gym.py

```diff
@@ -44,10 +44,23 @@ def test_render(all_scenarios):
 
     gym = ScenarioGym(timestep=0.1)
     gym.load_scenario(scenario_path)
-    gym.reset_scenario()
     gym.rollout(render=True)
+    gym.rollout(
+        render=True,
+        video_path=scenario_path.replace("Scenarios", "Recordings").replace(
+            ".xosc", "_2.mp4"
```

Comment:
why the _2?

Reply:
Thank you this was me debugging will fix.